ServerStatisticsPublisher/TopicPublisher.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Text;
+using Newtonsoft.Json;
+using RabbitMQ.Client;
+using Configurations;
+using ServerStatisticsService;
+
+namespace ServerStatisticsPublisher
+{
+    public static class TopicPublisher
```

Comment:
this publish and the consumer classes need to have interfaces, and config needed should be passed to the constructor instead of resolving the config in these classes.

Configs do not usually get changed on runtime. It would make sense to only resolve the config once from program, make it singleton, and pass it to the classes that needed it.